app/src/lib/stores/app-store.ts

```diff
@@ -4010,7 +4015,7 @@ export class AppStore extends TypedBaseStore<IAppState> {
 
       await fastForwardBranches(repository, eligibleBranches)
     } catch (e) {
-      log.error('Branch fast-forwarding failed', e)
+      log.error(`Branch fast-forwarding failed for ${repository.path} using codespace? ${repository.codespace}`, e)
```

Comment:
I need to track this error down...there's a weird bug where when we first add the "codespace" (current name) we go into a loop where this error shows up w/ the correct path but with the wrong value for `codespace`. The workaround is to add the codespace repo before starting the server...but that's a hacky workaround.